Client/game_sa/CSettingsSA.cpp

```diff
@@ -275,12 +275,13 @@ void CSettingsSA::Save()
 
 bool CSettingsSA::IsVolumetricShadowsEnabled()
 {
-    return m_bVolumetricShadowsEnabled && !m_bVolumetricShadowsSuspended;
+    return m_bVolumetricShadowsEnabled && !m_bVolumetricShadowsSuspended || m_bVolumetricShadowsEnabledByScript;
```

Comment:
Although `and` operator will be evaluated first, it would be better to surround the expression with parentheses.